src/api/exceptions.cc

```diff
@@ -232,7 +232,7 @@ Local<Value> WinapiErrnoException(Isolate* isolate,
   }
 
   if (must_free)
-    LocalFree((HLOCAL)msg);
+    LocalFree(HLOCAL(msg));
```

Comment:
This should probably also be an explicit cast?

Reply:
Like `reinterpret_cast<HLOCAL>(msg)` ? (I am not familiar with windows api, current one is suggested fix by VS 🙃 )

Reply:
@gengjiawen Yeah, I think so. :)

Reply:
Done :)